### ISSUE TITLE
Package installation fails on a new blank UWP app

### ISSUE THREAD

**Issue by karann-msft** [linvi/tweetinvi#337]
Here is the error from the output window:

```
System.Reflection.Emit.ILGeneration 4.0.1 provides a compile-time reference assembly for System.Reflection.Emit.ILGeneration on UAP,Version=v10.0, but there is no run-time assembly compatible with win10-arm-aot.
One or more packages are incompatible with UAP,Version=v10.0 (win10-arm-aot).
System.Reflection.Emit.ILGeneration 4.0.1 provides a compile-time reference assembly for System.Reflection.Emit.ILGeneration on UAP,Version=v10.0, but there is no run-time assembly compatible with win10-x64-aot.
One or more packages are incompatible with UAP,Version=v10.0 (win10-x64-aot).
System.Reflection.Emit.ILGeneration 4.0.1 provides a compile-time reference assembly for System.Reflection.Emit.ILGeneration on UAP,Version=v10.0, but there is no run-time assembly compatible with win10-x86-aot.
One or more packages are incompatible with UAP,Version=v10.0 (win10-x86-aot).
Package restore failed for 'App1'.
Package restore failed. Rolling back package changes for 'App1'.
```

**Comment by linvi** [comment]
Thank you for reporting the issue. This will be taken care of as soon as possible.
Could you install version 1.1.0.0 for now.

Thanks,
Linvi

**Comment by linvi** [comment]
I wanted to issue a fix by tonight but it is getting late here.
The problem is with nuget only, no code needs to be changed.

So you will have to wait tomorrow evening.
In the meantime please do use the version 1.0.

Sorry for the inconvenience.

**Comment by linvi** [comment]
I have just released version 1.1.0.4 which is just a new nuget package with the same binary but this time it should work with your W10 project.

Thank you.
